editoast/src/views/projection.rs

```diff
@@ -49,6 +53,16 @@ pub struct ProjectPathForm {
     pub track_section_ranges: Vec<TrackRange>,
 }
 
+#[derive(Debug, Deserialize, ToSchema)]
+pub struct ProjectPathOpForm {
+    pub infra_id: i64,
+    pub electrical_profile_set_id: Option<i64>,
+    pub ids: HashSet<i64>,
```

Comment:
We try to avoid fields named `id(s)`, for the API.

```suggestion
    pub train_ids: HashSet<i64>,
```